Doc/library/asyncio-eventloop.rst

```diff
@@ -215,21 +217,21 @@ Scheduling callbacks
    Schedule the *callback* :term:`callback` to be called with
    *args* arguments at the next iteration of the event loop.
 
-   Callbacks are called in the order in which they are registered.
-   Each callback will be called exactly once.
+   Callbacks are called in the order in which they are registered. Each callback
+   will be called exactly once, even if it is registered multiple times.
```

Comment:
It wasn't clear to me why it was important to note that 'each callback will be called exactly once' -- if I register a callback once, I would expect it only to be run once. 

My guess here is that a callback could be registered multiple times before execution loops back around to it, and in this case it's still called only once?

Alternatively -- I could see a design decision where a callback, once registered, is called repeatedly as the event loop cycles around, up until the point where it is _de-registered_ by some mechanism... but that would be a more surprising design to me.

Regardless: I suggest more clarity here around this "will be called exactly once".

Reply:
@gvanrossum Can you provide guidance on why this 'will be called exactly once' language is here?

Reply:
@bskinn
> if I register a callback once, I would expect it only to be run once.

That depends on the type of callback -- it is true here, but out of context that's a faulty assumption. E.g. `add_reader()` adds a callback that may be called multiple times.

Reply:
This change reverted entire by 7d18a15